e2e/keywords/volume.resource

```diff
@@ -86,6 +89,23 @@ Wait for volume ${volume_id} healthy
     ${volume_name} =    generate_name_with_suffix    volume    ${volume_id}
     wait_for_volume_healthy    ${volume_name}
 
+Wait for volume ${volume_id} degraded
+    ${volume_name} =    generate_name_with_suffix    volume    ${volume_id}
+    wait_for_volume_degraded    ${volume_name}
+
+Check volume ${volume_id} replica on node ${node_id} kept in stopped
+    ${volume_name} =    generate_name_with_suffix    volume    ${volume_id}
+    ${node_name} =    get_node_by_index    ${node_id}
+    FOR    ${i}    IN RANGE    ${LOOP_COUNT}
```

Comment:
It loops `30` times without `sleep`. How long will it wait before return?

Reply:
Updated.

Before update, the loop time in `Check volume ${volume_id} replica on node ${node_id} kept in stopped` was about 20s.